calico_containers/pycalico/util.py

```diff
@@ -71,3 +75,9 @@ def get_host_ips(version=4, exclude=None):
 
     return ip_addrs
 
+def get_hostname():
+    try:
+        hostname = os.environ[HOSTNAME_ENV]
+    except KeyError:
+        hostname = socket.gethostname()
```

Comment:
I don't have much input on design, but the more concise way to do this might be 
```
hostname = os.environ.get('HOSTNAME_ENV', socket.gethostname())
```
`os.environ.get` will return the default value if the key is not found